### ISSUE TITLE
Fix or temporarily disable the "notify-irc" GitHub workflow

### ISSUE THREAD

**Issue by tomhukins** [Perl/perl5#18329]
Recently, the "notify-irc" CI workflow has started failing with the following error:

```
Unable to process command '::set-env name=ref::' successfully.
Error: The `set-env` command is disabled. Please upgrade to using Environment Files or opt into unsecure command execution by setting the `ACTIONS_ALLOW_UNSECURE_COMMANDS` environment variable to `true`. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
```

We need to fix or temporarily disable this failing task to make it easier to spot legitimate problems with the build.  The use of `::set-env` was introduced in b4d4e4ebeba50be by @atoomic.

**Comment by atoomic** [comment]
Thanks I will look for a fix

**Comment by atoomic** [comment]
fixed by #18331
